src/HuntAndPeck/Services/HintLabelService.cs

```diff
@@ -40,7 +40,7 @@ public IList<string> GetHintStrings(int hintCount)
             }
 
             // Note that shuffle is lazy evaluated. Sigh.
```

Comment:
Minor, but don't need this comment any more. Shuffling was originally here to make it similar to Vimium/Vimperator